caddyconfig/httploader.go

```diff
@@ -45,6 +45,10 @@ type HTTPLoader struct {
 	// Maximum time allowed for a complete connection and request.
 	Timeout caddy.Duration `json:"timeout,omitempty"`
 
+	// The number of seconds to wait in between attempts to call `URL`.
+	// Defaults to 0, indicating no retries.
+	RetryDelay time.Duration `json:"retry_delay,omitempty"`
```

Comment:
```suggestion
	RetryDelay caddy.Duration `json:"retry_delay,omitempty"`
```

Not strictly necessary but is conventional as it allows users to specify `d` (day) units! Probably not relevant here but oh well.